Makefile

```diff
@@ -1,17 +1,28 @@
-PREFIX=/usr/
-DESTDIR=/
-BINARY_NAME=apx
+PREFIX := /usr
+DESTDIR := /
+BINARY_NAME := apx
+
+GO := go
+SYSTEM_DISTROBOX :=
 
 all: build
 
-build:
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -tags netgo -ldflags '-w -extldflags "-static"' -o ${BINARY_NAME}
+build: ${BINARY_NAME}
+
+${BINARY_NAME}:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 ${GO} build -a -tags netgo -ldflags '-w -extldflags "-static"' -o $@
 
+ifdef SYSTEM_DISTROBOX
 install: build
+else
+install: build install-distrobox
+endif
 	install -Dm755 ${BINARY_NAME} ${DESTDIR}${PREFIX}/bin/${BINARY_NAME}
 	mkdir -p ${DESTDIR}/etc/apx
-	sed -i 's|/usr/share/apx/distrobox|${PREFIX}/share/apx/distrobox|g' config/apx.json
+	sed -i 's|/usr/share/apx/distrobox|${DESTDIR}${PREFIX}/share/apx/distrobox|g' config/apx.json
```

Comment:
destdir is used for cases where the root differs, like when being packaged and it's installed into the package directory, this shouldn't be included in the config